packages/block-editor/src/hooks/font-size.js

```diff
@@ -150,6 +150,22 @@ export function useIsFontSizeDisabled( { name: blockName } = {} ) {
 	);
 }
 
+/**
+ * Given a font size slug preset, returns the matching preset.
+ *
+ * @param {string} slug The slug of the font size preset
+ * @return {object|undefined} The font size object, if found.
+ */
+export function useFontSizeFromPreset( slug ) {
```

Comment:
This is added to the public API. Should it be `__experimental`?

I'll add some tests and better comments if not.

Reply:
The `block-library` package uses [`lock` and `unlock`](https://github.com/WordPress/gutenberg/blob/887cf3dfc4b3426ce5696a383216186233c468e1/packages/block-library/src/lock-unlock.js#L6), so you could always make it private rather than public if you think it might change over time?